distribution/docs/src/main/resources/_contents/_draft-platform-contents/country-code-converter-contents.adoc

```diff
@@ -0,0 +1,32 @@
+==== Country Code Converter
+
+In the Platform Application, you have the option to install a feature called `platform-country-local`.
+This feature provides an OSGi service to convert country codes into various formats by performing a local lookup.
+
+In order to utilize the FIPS 10-4 to ISO 3166-1 alpha 3 and vice versa country code conversions, the `FIPS 10-4 to ISO 3166-1 alpha 3 File Location` field in the feature's configuration must be entered with the file path of a properties file containing the FIPS 10-4 to ISO 3166-3 alpha 3 country code mappings. The feature is already configured by default upon installation, with the file location field pointing to a properties file containing country code mappings from http://www.opengeocode.org/. This file can be found at: ${branding-lowercase}.home/etc/fipsToIso3.properties, and should only be modified in the case the country code mappings would ever change.
```

Comment:
Good point! I will update the documentation to explain that if the mappings file is updated the bundle will need to be restarted and then provide instructions on how to do that.